gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -109,6 +111,38 @@ const NxFormSelectPage = () =>
             </NxTable.Row>
           </NxTable.Body>
         </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>NxFormSelect CSS Classes</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Class</NxTable.Cell>
+              <NxTable.Cell>Location</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-form-select--short</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Directly on the <NxCode>NxFormSelect</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Short variant of <NxCode>NxFormSelect</NxCode>.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-form-select--long</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Directly on the <NxCode>NxFormSelect</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Long variant of <NxCode>NxFormSelect</NxCode>.</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
```

Comment:
This subsection has no header.  I don't think that's valid

Reply:
> This subsection has no header. I don't think that's valid

Actually, the contents are this subsection are actually supposed to be part of the first subsection, you've added the helper classes right in the middle of the state helpers section

Reply:
> > This subsection has no header. I don't think that's valid
> 
> Actually, the contents of this subsection are supposed to be part of the first subsection, you've added the helper classes right in the middle of the state helpers section

https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/9e8dd0cafb945be51b41fa96082a82532dd90e4e